CMakeLists.txt

```diff
@@ -1,13 +1,36 @@
-# This file is used for testing only
+project(editorconfig-core-qt)
+cmake_minimum_required(VERSION 3.20)
 
-# To perform the test, run `cmake .` at the root of the project tree followed
-# by ctest .
+set(CMAKE_AUTOMOC ON)
```

Comment:
This should be set to a target, no? IMHO, doing it globally in cmake 3.20+ is not a good idea.

Reply:
This library doesn't use `Q_OBJECT` macro anywhere so I don't think this would be required, and only needs set if a parent project needs it. Agree to remove this.

Reply:
bdec285